namespaces/live-1.cloud-platform.service.justice.gov.uk/nomis-api-access-staging/resources/rds.tf

```diff
@@ -2,7 +2,7 @@ variable "cluster_name" {}
 
 variable "cluster_state_bucket" {}
 
-module "nomis-api-access-staging_rds" {
+module "nomis-api-access_rds" {
```

Comment:
Changing the name of the module will re-create the RDS instance. You can check the [`plan`](https://concourse.cloud-platform.service.justice.gov.uk/teams/main/pipelines/plan-environments/jobs/plan-live-1/builds/259) for more detail. Are you happy to proceed with this?

Reply:
Yes that's fine.